src/main/java/org/broadinstitute/hellbender/utils/read/ReadUtils.java

```diff
@@ -457,6 +455,26 @@ public static int getReadCoordinateForReferenceCoordinateUpToEndOfRead(GATKRead
     }
 
     /**
+     * Returns the read coordinate corresponding to the requested reference coordinate.
+     *
+     * WARNING: if the requested reference coordinate happens to fall inside or just before a deletion (or skipped region) in the read, this function
+     * will return the last read base before the deletion (or skipped region). This function returns a
+     * Pair(int readCoord, boolean fallsInsideOrJustBeforeDeletionOrSkippedRegion) so you can choose which readCoordinate to use when faced with
+     * a deletion (or skipped region).
+     *
+     * SUGGESTION: Use getReadCoordinateForReferenceCoordinate(GATKSAMRecord, int, ClippingTail) instead to get a
+     * pre-processed result according to normal clipping needs. Or you can use this function and tailor the
+     * behavior to your needs.
+     *
+     * @param read
+     * @param refCoord the requested reference coordinate
+     * @return the read coordinate corresponding to the requested reference coordinate. (see warning!)
```

Comment:
Document the meaning of the `Pair<Integer, Boolean>` return value in the `@return` annotation